packages/components/src/Enumeration/Items/Item/Item.component.js

```diff
@@ -48,7 +48,7 @@ function Item({ id, item, searchCriteria, showCheckboxes, style, t }) {
 			}
 		}
 
-		return (
+		return !action.disabled ? (
```

Comment:
you can add a test on disabled and if it is the case return null.
add if(action.disabled) { return null }